### PR TITLE
Update gulpfile.js to include sample imports

### PR DESCRIPTION
**Pull request by krumware**
Ref #65
Update gulpfile.js to include sample imports for gulp-uglify, css-slam, and gulp-html-minifier.
Also update comment in usage of gulp-html-minifier to reference the proper package name and eliminate confusion.

### CODE REVIEW

**Comment by jsilvermist** [comment]
I believe the intended purpose was simple to show where additional steps should be added to the build process, not to add in these ones specifically.

People could easily mistake this to work simply by uncommenting, not realizing they would also have to install the packages for it to work.

It would at least need commenting to either:
1. Run `npm install --save-dev ???` for each package.
1. Add `???` to `devDependencies` in `package.json` and run `npm install`.

**Comment by krumware** [comment]
I agree that there some addition instruction would be helpful, but didn't want to go too far.  My goal here was to bring the gulpfile a little bit closer to the example provided in "Handling Inlined CSS/JS" in the polymer-build docs (https://github.com/Polymer/polymer-build#handling-inlined-cssjs) to try to eliminate some ambiguity.

I think the instructions for installation (and description) of the packages belongs more in the polymer-build docs than here, but this template serves as a great example for implementation.

I can write some clear notes inline, but would prefer to do that in a separate pull req. Do you think it would be better to educate the reader directly with installation instructions, or refer them to specific sections of the polymer-build readme, or perhaps both?

**Comment by jsilvermist** [comment]
I was thinking of something on the imports like:
```js
// Not installed by default, before use run `npm install --save-dev <package-name>`
```
and maybe separating them slighting to set them apart from what is installed.

**Comment by robdodson** [comment]
+1 to @jsilvermist's suggestion

**Comment by krumware** [comment]
Sounds good guys,  I'll make the modifications this morning 👍

**Comment by krumware** [comment]
Hi guys, ready for additional feedback 👍

File: generators/app/gulpfile.js [hunk 1]
```diff
@@ -13,13 +13,18 @@
...
-//const uglify = require('gulp-uglify');
-//const cssSlam = require('css-slam').gulp;
-//const htmlMinifier = require('gulp-html-minifier');
 const mergeStream = require('merge-stream');
 const polymerBuild = require('polymer-build');
 
+// Here we add tools that will be used to process our source files.
+const imagemin = require('gulp-imagemin');
+
+// Additional plugins can be used to further optimize our source files after splitting.
```
Comment (robdodson) [inline generators/app/gulpfile.js]: tiny nit, can you tweak this to say "optimize your source files"

**Comment by robdodson** [comment]
Great thanks!

**Pull request merged**